server/readme.md

```diff
@@ -25,8 +25,8 @@ The following environment variables are required:
     {
```

Comment:
To me it makes more sense to provide a sample `.env` rather than have the vcaps in the readme.